synapse/events/__init__.py

```diff
@@ -31,7 +31,7 @@ def get_dict(self):
         return dict(self.__dict__)
 
     def is_outlier(self):
-        return hasattr(self, "outlier") and self.outlier
+        return getattr(self, "outlier", False)
```

Comment:
This isn't strictly relevant to the PR, but I came across it when trying to see how we decide when to store a state group.